packages/emsdk/build.sh

```diff
@@ -0,0 +1,17 @@
+TERMUX_PKG_HOMEPAGE=https://emscripten.org/
+TERMUX_PKG_DESCRIPTION="Emscripten is a complete compiler toolchain to WebAssembly"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_VERSION=2.0.8
+TERMUX_PKG_SRCURL=https://github.com/emscripten-core/emsdk/archive/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=248d716dbe2e2424ee3f205c41024b4efde974e8987f2204a97edb315c588ff3
+TERMUX_PKG_DEPENDS="llvm"
+TERMUX_PKG_BUILD_DEPENDS="python3, node, java"
```

Comment:
Please note that build depends specify only Termux packages. Not packages of the host system. Also - one of specified package names available in Termux.

Don't forget that you cross-compiling package not only for different arch but for different platform.

Reply:
```
ERROR: Package emsdk depends on non-existing package "python3"
ERROR: Obtaining buildorder failed
Error: Process completed with exit code 1.
```